adr/choose_npm_publish.md

```diff
@@ -0,0 +1,42 @@
+---
+category: adrs
+expires: 2020-11-05
+---
+
+# Choose NPM Publish
+
+## Status
+
+Accepted
+
+## Context
+
+As we split our code in microservices on the front-end, we need to have a way of sharing common code across products.
+
+We have this issue currently with a few features we use over many products. This will give developers an option to share these features with others.
+
+## Decision
+
+Where possible, we should use NPM to publish any reusable code in our projects and consume them.
+
+We already use NPM repositories for all our products. Integration isn't an issue when consuming the packages.
```

Comment:
Phrasing here might be better as "we already use NPM repos within our products for dependencies"? 

Also do we want to talk NPM vs Yarn and potential consequences down the line?

Reply:
Am rewording this now but the NPM vs Yarn I feel is outside of the scope. However I think mentioning consuming via NPM or Yarn somewhere to clarify explicitly that it is about the repository use and not the tooling.

Reply:
Cleared the wording up on this